test/automation/src/sql/notebook.ts

```diff
@@ -104,7 +104,7 @@ export class Notebook {
 	public async selectAllTextInEditor(): Promise<void> {
 		const editor = '.notebook-cell.active .monaco-editor';
 		await this.code.waitAndClick(editor);
-		await this.code.dispatchKeybinding('ctrl+a');
+		await this.code.dispatchKeybinding('cmd+a');
```

Comment:
Can we support both Mac and other platforms here?

Reply:
Yep we can, I just kept is as cmd for now since we only run smoke tests on mac but I can change it in a future PR.